user-registration-app/src/app/services/resolvers/TableAssignmentResolver/table-assignment.resolver.ts

```diff
@@ -3,24 +3,24 @@ import { Observable } from 'rxjs';
 import { AppConstants } from '../../../AppConstants';
 import { finalize, map, mergeMap, take } from 'rxjs/operators';
 import { AuthService } from '../../AuthService/auth.service';
-import { NgProgress } from '@ngx-progressbar/core';
+import { NgProgress } from 'ngx-progressbar';
 import { HttpService } from '../../HttpService/HttpService';
 import { Injectable } from '@angular/core';
-import { ProjectModel } from "../../../models/project-model";
-import { AlertService } from "ngx-alerts";
+import { ProjectModel } from '../../../models/project-model';
+import { ToastrService } from 'ngx-toastr';
 
 @Injectable()
 export class TableAssignmentResolver implements Resolve<ProjectModel> {
   constructor(private authService: AuthService,
               private progress: NgProgress,
               private httpService: HttpService,
               private router: Router,
-              private alertService: AlertService,
+              private toastrService: ToastrService,
   ) {
   }
 
   resolve(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<ProjectModel> {
-    this.alertService.danger('Table assignment is broken. Please use Devpost instead');
+    this.toastrService.error('Table assignment is broken. Please use Devpost instead');
```

Comment:
Assuming we're always going to be using devpost is there a reason this file can't just be deleted? especially seeing as though it's broken and I'm not sure it's worth it to create/use our own implementation over devpost's

Reply:
Just nuked 🎉